shuup/front/utils/sorts_and_filters.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+
+import abc
+
+import six
+from django import forms
+from django.conf import settings
+from django.db.models import Q
+
+from shuup import configuration
+from shuup.apps.provides import get_provide_objects
+
+FACETED_DEFAULT_CONF_KEY = "front_faceted_configurations"
+FACETED_CATEGORY_CONF_KEY_PREFIX = "front_faceted_category_configurations_%s"
+FORM_MODIFIER_PROVIDER_KEY = "front_extend_product_list_form"
+
+
+class ProductListFormModifier(six.with_metaclass(abc.ABCMeta)):
+    def should_use(self, configuration):
+        """
+        :param configuration: current configurations
+        :type configuration: dict
+        :return: Boolean whether the modifier should be used based
+        on current configurations.
+        :rtype: boolean
+        """
+
```

Comment:
Extra line change.

Also you could explain where this affects. If this is in `should_use = True` where do I see some changes, what does this affect? What is this `ProductListFormModifier`?